src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests2.cs

```diff
@@ -1093,15 +1091,15 @@ public static void M(object o)
     }
 }
 ";
-            var compilation = CreatePatternCompilation(source);
+            var compilation = CreateCompilationWithMscorlib45(source); // doesn't have ITuple
```

Comment:
>doesn't have ITuple [](start = 74, length = 19)

Would this test behave differently if the ITuple type was found?

Reply:
Yes; compilation would pass and it would use `ITuple` for the matching.

Reply:
Sorry, I didn't follow. The parameter `object o` doesn't implement `ITuple`.

---
In reply to: [205628798](https://github.com/dotnet/roslyn/pull/28859#discussion_r205628798) [](ancestors = 205628798)